akka-http-core/src/main/resources/reference.conf

```diff
@@ -404,4 +404,6 @@ akka.http {
     # HttpResponse on client side).
     tls-session-info-header = off
   }
+
+  strict-entity-max-bytes = 9223372036854775807
```

Comment:
I suggest to express the value using a unit format, for example `8192 pebibytes` as documented here https://github.com/typesafehub/config/blob/master/HOCON.md#size-in-bytes-format